package.json

```diff
@@ -9,7 +9,8 @@
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
     "biome": "biome check --error-on-warnings --write",
-    "prettier": "prettier . --write --plugin=prettier-plugin-tailwindcss"
+    "prettier:check": "prettier . --check",
```

Comment:
`--check` option seems to be useful when used in CI

> The command will return exit code `1` in the second case, which is helpful inside the CI pipelines.

https://prettier.io/docs/en/cli#--check